### PR TITLE
created the basic template for the main page

### PR DESCRIPTION
**Pull request by iamAravindks**

### CODE REVIEW

File: todaythediary/debug.log [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[1102/165530.511:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (anandbaburajan) [inline todaythediary/debug.log:1]: Can you ignore these files?

**Review by anandbaburajan — CHANGES_REQUESTED** [review]
@iamAravindks the code looks good! I've added one comment. I didn't run the app yet: will do once more changes are added!

File: todaythediary/debug.log (hunk 1 above)
Comment (iamAravindks) [reply todaythediary/debug.log:1]: @AnandBaburajan did you mean by ignoring the debug.log file? Can you specify more about this, please?

Comment (anandbaburajan) [reply todaythediary/debug.log:1]: > @AnandBaburajan did you mean by ignoring the debug.log file? Can you specify more about this, please?

Aaah, yes. That's what I meant. Remove the file.

**Pull request merged**